docs/topics/health-check-mission-basic-interaction-spring-boot-tomcat.adoc

```diff
@@ -33,10 +33,19 @@ $ curl http://{app-name}-{project-name}.{os-route-hostname}/api/stop
 $ curl http://{app-name}-{project-name}.{os-route-hostname}/api/greeting
 
 <html>
-  <head><title>Error</title></head>
-  <body>503 - Service Unavailable</body>
+  <head>
+  ...
+  </head>
+  <body>
+    <div>
+      <h1>Application is not available</h1>
+      ...
+    </div>
+  </body>
 </html>
 ----
++
+NOTE: Depending on when OpenShift removes the pod after you invoke the /api/stop endpoint, you may initially see a 404 error code. If continue to invoke the /api/greeting endpoint, you will see the `Application is not available` page after OpenShift removes the pod.
```

Comment:
_/api/stop_ and _/api/greeting_ should be verbatim, i. e. surrounded by backticks.

Reply:
was keeping consistency with the topic, but agreed. Fixing in the whole file.